modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -30,21 +35,29 @@ spec:
     type: eBPF
     ebpf:
       flowFilter:
-        action: Accept  <1>
-        cidr: 172.210.150.1/24 <2>
-        protocol: SCTP
-        direction: Ingress
-        destPortRange: 80-100
-        peerIP: 10.10.10.10
-        enable: true    <3>
+        enable: true <1>
+        rules:
+         - action: Accept <2>
+           cidr: 0.0.0.0/0 <3>
+           sampling: 1 <4>
+         - action: Accept
+           cidr: 10.128.0.0/14
+           peerCIDR: 10.128.0.0/14 <5>
+         - action: Accept
+           cidr: 172.30.0.0/16
+           peerCIDR: 10.128.0.0/14
+           sampling: 50
 ----
-<1> The required `action` parameter describes the action that is taken for the flow filter rule. Possible values are `Accept` or `Reject`. 
-<2> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` for IPv4 or `::/0` for IPv6.
-<3> You must set `spec.agent.ebpf.flowFilter.enable` to `true` to enable this feature.
+<1> To enable eBPF flow filtering, you must set `spec.agent.ebpf.flowFilter.enable` to `true`.
+<2> The required `action` parameter specifies the action that is taken for the flow filter rule. Possible values are `Accept` or `Reject`.
+<3> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` for IPv4 or `::/0` for IPv6.
+<4> Sampling rate for the matched flows, overriding the global sampling defined at `spec.agent.ebpf.sampling`.
+<5> The Peer IP CIDR to filter flows by.
++
+By default, all other flows are rejected.
```

Comment:
If possible, the items in this list of callouts should have more parallel phrasing. Right now we have:
* To enable...
* The [XYZ] parameter specifies...
* The [ABC] parameter provides...
* Sampling rate... (sentence fragment)
* The [description of parameter function]...